src/components/nowplayingbar/nowplayingbar.js

```diff
@@ -54,7 +54,7 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
 
         html += '<div class="nowPlayingBarRight">';
 
-        html += '<button is="paper-icon-button-light" class="muteButton mediaButton"><i class="material-icons volume_up"></i></button>';
+        html += '<button is="paper-icon-button-light" class="muteButton mediaButton"><i class="material-icons"></i></button>';
```

Comment:
I think a better fix would be performed here: https://github.com/jellyfin/jellyfin-web/blob/3b5a63f930b4452dafd04bce57b5875f19c2cece/src/components/nowplayingbar/nowplayingbar.js#L416

Seems like that needs to be changed to remove the volume_up class from the `<i>` and add the other corresponding class.

Reply:
yap that's why i remove those class b/c it will add  it <i>-here-</i>